medusa/name_parser/parser.py

```diff
@@ -224,6 +224,10 @@ def _parse_string(self, name):
 
         return result
 
+    def unparse(self, indexer, indexer_id):
```

Comment:
'Unparse' doesn't make any sense. Better to just use clear_parsed_name_cache(). Or something similar.